index.html

```diff
@@ -115,7 +116,7 @@ <h2>Introduction</h2>
         services. While this architecture can be used in the service of better Web experiences,
         it can also be abused to violate privacy ([[?privacy-principles]]). While data can be shared
         with service providers for limited operational purposes, it can also be shared with third
-        parties or used for behavioral targeting in ways that many users find objectionable.
+        parties or used for [=cross-organization ad targeting=] in ways that many users find objectionable.
```

Comment:
```suggestion
        parties in ways that many users find objectionable.
```
Dropping targeting discussion here to not jumble a narrative that doesn't even mention sale either.